scripts/cspell/project-words.txt

```diff
@@ -23,7 +23,7 @@ datacenter
 datasource
 developercertificate
 dilligence
-distirbuted
+distributed
```

Comment:
I don't believe we need this in the words list. btw dilligence is also misspelled (only needs one l)